token/cli/src/config.rs

```diff
@@ -117,18 +137,8 @@ impl<'a> Config<'a> {
         } else {
             solana_cli_config::Config::default()
         };
-        let multisig_signers = signers_of(matches, MULTISIG_SIGNER_ARG.name, wallet_manager)
-            .unwrap_or_else(|e| {
-                eprintln!("error: {}", e);
-                exit(1);
-            });
-        if let Some(mut multisig_signers) = multisig_signers {
-            multisig_signers.sort_by(|(_, lp), (_, rp)| lp.cmp(rp));
-            let (signers, pubkeys): (Vec<_>, Vec<_>) = multisig_signers.into_iter().unzip();
-            bulk_signers.extend(signers);
-            multisigner_ids.extend(pubkeys);
-        }
-        let multisigner_pubkeys = multisigner_ids.iter().collect::<Vec<_>>();
+        let multisigner_pubkeys =
+            Self::extract_multisig_signers(matches, wallet_manager, bulk_signers, multisigner_ids);
```

Comment:
This just a little movement that I thought would be needed to test, and then it wasn't, but it was a neat little refactor so I kept it